### DISCUSSION TITLE
Can't build project on simulator

### DISCUSSION THREAD

**Discussion by manvelhovhannisyan77** [LeoNatan/LNPopupController#510] [General] [answered]
On Xcode: 13.2.1, Chip: apple M1 pro

Could not find module 'LNPopupController' for target 'x86_64-apple-ios-simulator'; found: arm64, arm64-apple-ios-simulator, at: /Users/[userName]/Library/Developer/Xcode/DerivedData/[targetName]-epxnvixbxxnkukevqlsvuocmvqwd/Build/Products/Debug-iphonesimulator/LNPopupController.swiftmodule

I think the fix should be done on library development side. Please find solution below

The fix on cocoapods libs
post_install do |installer|
  installer.pods_project.build_configurations.each do |config|
    config.build_settings["EXCLUDED_ARCHS[sdk=iphonesimulator*]"] = "arm64"
  end
end

**Comment by LeoNatan** [comment, marked as answer]
You are using an unofficial release. LNPopupController does not support CocoaPods.
